src/web/components/Core/InlineError.scss

```diff
@@ -0,0 +1,11 @@
+.inline-error {
+  margin: 0 8px;
+}
+
+.inline-error-content {
```

Comment:
Looking at Figma, the inline error example has a font color (#030A40;) that I don't think is being applied here.